### PR TITLE
add new websites (to comply with the new api)

### PR DESCRIPTION
**Pull request by Brandonbr1**

### CODE REVIEW

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2025]: does not redistribute, links to original

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2049]: duplicate

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2045]: duplicate

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2041]: does not redistr

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2029]: duplicate

File: lists/minecraft.yaml
Comment (dada513) [inline lists/minecraft.yaml:2033]: im not sure about mcpedl, can you give your feedback on #426?

**Review by dada513 — CHANGES_REQUESTED** [review]
Also, list should be sorted alphabetically. Once you're done with my review I can sort it before merge

**Comment by Brandonbr1** [comment]
@dada513  did all the changes you asked  me to

**Comment by dada513** [comment]
> @dada513 did all the changes you asked me to

waiting for @berrysauce feedback on mcpdl

**Review by berrysauce — CHANGES_REQUESTED** [review]
Remove mcpedl.com for now. We need more time to review/discuss it.

**Pull request merged**
